app/src/org/commcare/engine/cases/CaseUtils.java

```diff
@@ -64,11 +64,7 @@ public static void purgeCases() {
         }
 
         SQLiteDatabase db;
-        try {
-            db = CommCareApplication._().getUserDbHandle();
-        } catch (SessionUnavailableException e) {
-            throw new UserStorageClosedException(e.getMessage());
```

Comment:
I've removed `UserStorageClosedException` because it was just a `RuntimeException` front for `SessionUnavailableException`